src/subapps/admin/components/Settings/DangerZoneSubView.tsx

```diff
@@ -246,7 +345,7 @@ const DangerZoneSubView = ({ project }: Props) => {
         <h2>Danger Zone</h2>
         <div className="settings-view-container">
           <div className="danger-actions">
-            <List<TDangerZoneItem>
+            <List<DangerZoneItem>
```

Comment:
can we keep previous types and names of any vars as it is for the moment until we prepare some naming/typing convention, it's not right to rename as everyone prefer, 
as long as the naming do not has typos, will keep it

Reply:
Reverted it.